ui/apps/platform/src/Containers/Clusters/Components/ClusterDeletion.tsx

```diff
@@ -24,19 +24,23 @@ function ClusterDeletion({ clusterRetentionInfo }: ClusterDeletionProps): ReactE
         const { daysUntilDeletion } = clusterRetentionInfo;
         const healthStatus = getClusterDeletionStatus(daysUntilDeletion);
         const { bgColor, fgColor } = healthStatusStyles[healthStatus];
-        const text = daysUntilDeletion === 1 ? 'in 1 day' : `in ${daysUntilDeletion} days`;
+        /* eslint-disable no-nested-ternary */
+        const text =
+            daysUntilDeletion < 1
+                ? 'Imminent'
+                : daysUntilDeletion === 1
+                ? 'in 1 day'
+                : `in ${daysUntilDeletion} days`;
```

Comment:
I personally think a `switch` is easier to read than nested ternary, but 1 extra level, as it is here, it's ok.